pandas/io/gbq.py

```diff
@@ -106,13 +107,14 @@ class TableCreationError(PandasError, ValueError):
     """
     pass
 
-
 class GbqConnector(object):
 
-    def __init__(self, project_id, reauth=False):
+    def __init__(self, project_id, reauth=False, private_key_json_path=None):
         self.test_google_api_imports()
```

Comment:
thanks for feedback on my change

* precisely which doc-string  ?
* `private_key_json_path` param is inteded to containt json file path only. Its name is self explaining. Google console gives service account private key file as json, so it is convenient to store it locally and point file location to gbq. Passing full contents of this file will be impractical.